source/fem-dem/vans_assemblers.cc

```diff
@@ -665,25 +668,25 @@ GLSVansAssemblerShearForce<dim>::calculate_particle_fluid_interactions(
 
   // Viscosity
   const double viscosity = physical_properties.viscosity;
-  const double density   = physical_properties.density;
 
   // Loop over particles in cell
   for (auto &particle : pic)
     {
       auto particle_properties = particle.get_properties();
 
-      // Undisturbed Flow Force
+      // Shear Force
       shear_force =
         -(M_PI * pow(particle_properties[DEM::PropertiesIndex::dp], dim) /
           (2 * dim)) *
-        density * viscosity * velocity_laplacians[particle_number];
+        viscosity * velocity_laplacians[particle_number];
 
       for (int d = 0; d < dim; ++d)
         {
           particle_properties[DEM::PropertiesIndex::fem_force_x + d] +=
             shear_force[d];
           undisturbed_flow_force[d] +=
-            shear_force[d] / particle_properties[DEM::PropertiesIndex::mass];
+            shear_force[d] /
+            (physical_properties.density * scratch_data.cell_volume);
```

Comment:
the shear force and the pressure force already have the right units, you don't need to divide by density because your viscosity is already divided by density (nu instead of mu).